.github/workflows/ios-release.yml

```diff
@@ -9,41 +21,41 @@ on:
   workflow_call:
     inputs:
       tag:
-        description: 'Tag/version (ex: 5.0.1)'
+        description: 'The existing tag (version) to be released (ex: 5.0.1).'
         required: true
         type: string
 
-      run_full_release:
-        description: 'Run full release process (`no` to skip)'
-        required: true
-        default: 'yes'
-        type: string
+      full_release_extensions:
+        description: |
+          A comma-separated list of extensions for which to run the full release process.
 
-      release_pod:
-        description: 'Release to Cocoapods'
-        required: true
-        default: 'yes'
-        type: string
+          **Syntax:** `"Extension1, Extension2, Extension3"`
 
-      library_name:
-        description: 'Library name to be released (ex: AEPEdge)'
-        required: true
-        type: string
-      
-      # Makefile environment variables
-      scheme_name_xcframework:
-        description: 'Scheme name for XCFramework (ex: AEPEdgeXCF)'
+          **Example:** `"AEPServices, AEPCore"`
         required: true
+        default: ''
         type: string
 
-      test_app_ios_scheme:
-        description: 'iOS Test App Scheme (ex: TestAppiOS)'
-        required: true
+      extension_dependencies:
+        description: |
```

Comment:
I would prefer a better name for this. Can you also clarify the description, as this is a Core-only requirement (since we bundle AEPRulesEngine xcframeworks as part of Core releases)? Other extensions shouldn’t use it.